### PR TITLE
wkhtmltopdf: add page

### PR DESCRIPTION
**Pull request by parth93QA**
Adding basename command in tldr for linux

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by acuteenvy — COMMENTED** [review]
A page for `basename` [already exists in `common`](https://github.com/tldr-pages/tldr/blob/main/pages/common/basename.md) with very similar examples.

**Comment by parth93QA** [comment]
adding wkhtmltopdf.md @acuteenvy  and removed basename.md

File: pages/common/wkhtmltopdf.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# wkhtmltopdf
+
+> It is an open-source command-line tool that allows you to convert HTML documents or web pages into PDF files.
+> It is primarily a command-line tool, which means you use it by running commands in a terminal or shell.
+> More information: <https://wkhtmltopdf.org/>.
+
+- Basic Conversion:
+
+`wkhtmltopdf input.html output.pdf`
```
Comment (blueskyson) [inline pages/common/wkhtmltopdf.md:9]:
```suggestion
`wkhtmltopdf {{input.html}} {{output.pdf}}`
```

File: pages/common/wkhtmltopdf.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Specify Page Size:
+
+`wkhtmltopdf --page-size A4 input.html output.pdf`
```
Comment (blueskyson) [inline pages/common/wkhtmltopdf.md:13]:
```suggestion
- Specify Page Size (please see `PaperSize` of `QPrinter` for supported sizes):

`wkhtmltopdf --page-size {{A4}} {{input.html}} {{output.pdf}}`
```

File: pages/common/wkhtmltopdf.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Set Margins:
+
+`wkhtmltopdf --margin-top 10mm --margin-bottom 10mm --margin-left 10mm --margin-right 10mm input.html output.pdf`
```
Comment (blueskyson) [inline pages/common/wkhtmltopdf.md:17]:
```suggestion
`wkhtmltopdf --margin-{{top|bottom|left|right}} {{10mm}} {{input.html}} {{output.pdf}}`
```

File: pages/common/wkhtmltopdf.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Custom Page Orientation:
+
+`wkhtmltopdf --orientation Landscape input.html output.pdf`
```
Comment (blueskyson) [inline pages/common/wkhtmltopdf.md:21]:
```suggestion
`wkhtmltopdf --orientation {{Landscape|Portrait}} {{input.html}} {{output.pdf}}`
```

File: pages/common/wkhtmltopdf.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+- Generate Grayscale PDF:
+
+`wkhtmltopdf --grayscale input.html output.pdf`
```
Comment (blueskyson) [inline pages/common/wkhtmltopdf.md:25]:
```suggestion
`wkhtmltopdf --grayscale {{input.html}} {{output.pdf}}`
```

**Review by blueskyson — COMMENTED** [review]
Thanks for contribution, here are some suggestions.

**Comment by parth93QA** [comment]
@blueskyson Changes are applied

**Review by blueskyson — APPROVED** [review]

File: pages/common/wkhtmltopdf.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# wkhtmltopdf
+
+> An open-source command-line tool to convert HTML documents or web pages into PDF files.
+> More information: <https://wkhtmltopdf.org/>.
+
+- Basic Conversion:
+
+`wkhtmltopdf {{input.html}} {{output.pdf}}`
+
+- Specify Page Size (please see `PaperSize` of `QPrinter` for supported sizes):
```
Comment (reinhart1010) [inline pages/common/wkhtmltopdf.md:10]:
```suggestion
- Specify the PDF page size (please see `PaperSize` of `QPrinter` for supported sizes):
```

File: pages/common/wkhtmltopdf.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+`wkhtmltopdf --page-size {{A4}} {{input.html}} {{output.pdf}}`
+
+- Set Margins:
```
Comment (reinhart1010) [inline pages/common/wkhtmltopdf.md:14]:
```suggestion
- Set the PDF page margins:
```

File: pages/common/wkhtmltopdf.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`wkhtmltopdf --margin-{{top|bottom|left|right}} {{10mm}} {{input.html}} {{output.pdf}}`
+
+- Custom Page Orientation:
```
Comment (reinhart1010) [inline pages/common/wkhtmltopdf.md:18]:
```suggestion
- Set the PDF page orientation:
```

File: pages/common/wkhtmltopdf.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# wkhtmltopdf
+
+> An open-source command-line tool to convert HTML documents or web pages into PDF files.
+> More information: <https://wkhtmltopdf.org/>.
+
+- Basic Conversion:
```
Comment (reinhart1010) [inline pages/common/wkhtmltopdf.md:6]:
```suggestion
- Convert a HTML document into PDF:
```

Sentence-case and more descriptive.

File: pages/common/wkhtmltopdf.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+- Set Margins:
+
+`wkhtmltopdf --margin-{{top|bottom|left|right}} {{10mm}} {{input.html}} {{output.pdf}}`
+
+- Custom Page Orientation:
+
+`wkhtmltopdf --orientation {{Landscape|Portrait}} {{input.html}} {{output.pdf}}`
+
+- Generate Grayscale PDF:
```
Comment (reinhart1010) [inline pages/common/wkhtmltopdf.md:22]:
```suggestion
- Generate a greyscale version of the PDF document:
```

Note that as of now [our style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) doesn't specify which version of English (American or British) these pages should be written in.

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Hi, here are some of my review notes. In general, we have a convention to write command description in sentence case (like [`common/cd`](https://nix.reinhart1010.id/en/common/cd.html) and [`common/xetex`](https://nix.reinhart1010.id/en/common/xetex.html)), though it is not currently written in our style guide.

**Comment by parth93QA** [comment]
Hi @reinhart1010 I have made the requested changes.

File: pages/common/wkhtmltopdf.md (hunk 10 above)
Comment (parth93QA) [reply pages/common/wkhtmltopdf.md]: Done @reinhart1010

File: pages/common/wkhtmltopdf.md (hunk 9 above)
Comment (parth93QA) [reply pages/common/wkhtmltopdf.md]: Done @reinhart1010

File: pages/common/wkhtmltopdf.md (hunk 8 above)
Comment (parth93QA) [reply pages/common/wkhtmltopdf.md]: Done @reinhart1010

File: pages/common/wkhtmltopdf.md (hunk 7 above)
Comment (parth93QA) [reply pages/common/wkhtmltopdf.md]: Done @reinhart1010

File: pages/common/wkhtmltopdf.md (hunk 6 above)
Comment (parth93QA) [reply pages/common/wkhtmltopdf.md]: Done @reinhart1010

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Pull request merged**